### ISSUE TITLE
Repository Bug

### ISSUE THREAD

**Issue by itsmessc** [microsoft/devhome#2628]
### Dev Home version

0.1201.442.0

### Windows build number

10.0.22631.3447

### Other software

OS Build Version: 10.0.22631.3447.amd64fre.ni_release.220506-1250
.NET Version: .NET 8.0.1

### Steps to reproduce the bug

After I pressed clone repository and add repository

### Expected result

It should ask which repo should be cloned

### Actual result

The app crashes and closed

### Included System Information

CPU: AMD Ryzen 5 4600H with Radeon Graphics         
Physical Memory: 7.37GB (1.13GB free)
Processor Architecture: x64

### Included Extensions Information

Extensions: 
Microsoft.Windows.DevHome_0.1201.442.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHome_0.1201.442.0_x64__8wekyb3d8bbwe
Microsoft.DevHomeMicrosoftGameDevExtension_0.400.456.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHomeGitHubExtension_0.1200.442.0_x64__8wekyb3d8bbwe

Widget Service: 
MicrosoftWindows.Client.WebExperience_424.1301.450.0_x64__cw5n1h2txyewy

**Comment by raphaelgpalma** [comment]
I encountered the same error, and I chose to monitor the Event Viewer to gather more insights. Here's the event log I obtained:

Event Viewer output:

Faulting application name: DevHome.exe, version: 1.0.0.0, time stamp: 0x65680000
Faulting module name: Microsoft.UI.Xaml.dll, version: 3.1.5.0, time stamp: 0x678b2a6f
Exception code: 0xc000027b
Fault offset: 0x0000000000404005
Faulting process id: 0x0x2BF0
Faulting application start time: 0x0x1DA8DE57B3F680E
Faulting application path: C:\Program Files\WindowsApps\Microsoft.Windows.DevHome_0.1201.442.0_x64__8wekyb3d8bbwe\DevHome.exe
Faulting module path: C:\Program Files\WindowsApps\Microsoft.WindowsAppRuntime.1.5_5001.95.533.0_x64__8wekyb3d8bbwe\Microsoft.UI.Xaml.dll
Report Id: 8f362ff6-4261-4a8f-956c-eb565476516f
Faulting package full name: Microsoft.Windows.DevHome_0.1201.442.0_x64__8wekyb3d8bbwe
Faulting package-relative application ID: App

It appears that this is a problem related to DevHome.exe, with the Microsoft.UI.Xaml.dll module causing a crash, maybe because user interface. I'll continue investigating and will update you with any progress.

System Information:

CPU: AMD Ryzen 5 5600 
GPU: NVIDIA GTX 1660 SUPER
Physical Memory: 15.9GB (9.5GB free)
Processor Architecture: x64
